charts/agent/templates/_helpers.tpl

```diff
@@ -242,4 +242,20 @@ Agent agentConfigmapName
 */}}
 {{- define "agent.configmapName" -}}
     {{- default .Values.global.agentConfigmapName | default "sysdig-agent" -}}
+{{- end -}}
+
+{{/*
+Use global sysdig tags for agent
+*/}}
+{{- define "sysdig.settings.tags" -}}
```

Comment:
Nit: This is probably fine, but given that we also have a value with this name, maybe we can call this something simple like `agent.tags`